### PR TITLE
Setup GTFS data and Bounding box

### PR DESCRIPTION
**Pull request by MTRNord**
This are all changes I did so far except the changes to the nomalizing. This should get the thing in a state where it can be directly deployed :)

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -4,8 +4,8 @@ set -e
 set -o pipefail
 set -x
 
-# todo: find a URL that always points to the latest dataset
-wget -O gtfs.zip 'todo'
+#wget -O gtfs.zip 'http://www.connect-info.net/opendata/gtfs/nah.sh/%CONNECT-INFO-TOKEN%'
+wget -O gtfs.zip 'https://gtfs.mfdz.de/NAH.SH.raw.gtfs.zip'
```
Comment (derhuerst) [inline build.sh:11]:
```suggestion
# gtfs.mfdz.de mirrors the Nah.SH GTFS dataset without a token
wget -O gtfs.zip 'https://gtfs.mfdz.de/NAH.SH.raw.gtfs.zip'
```

File: package.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
 	],
 	"author": [
 		"Jannis R <mail@jannisr.de>",
-		"Holger Bruch <holger.bruch@systect.de>",
+		"Holger Bruch <holger.bruch@systect.de>"
```
Comment (derhuerst) [inline package.json]: Add yourself if you want!

File: package.json [hunk 3]
```diff
@@ -33,10 +33,10 @@
 		"node": ">=14"
 	},
 	"dependencies": {
-		"cached-hafas-client": "^4.0.0",
```
Comment (derhuerst) [inline package.json]: The `^` in the value expresses a SemVer range, so this change is not relevant. (The actual version of a dependency being installed is determined by the lockfile if present.)

In fact, non-major-upgrades are a bit like noise, e.g. when I want to `git blame`, so please remove this. Thanks!

**Review by derhuerst — COMMENTED** [review]
Looks good, just some minor things!

File: package.json (hunk 3 above)
Comment (MTRNord) [reply package.json]: Fixed by removing the commit where I changed that. So it doesnt clutter up the history

**Pull request merged**